### DISCUSSION TITLE
Daylight Savings time

### DISCUSSION THREAD

**Discussion by Steve-Hight** [chrisblakley/Daylight-Calendar-ICS#15] [General]
Hi Chris, thank you for this calendar.  Does the calendar auto adjust for daylight saving time?  I am especially interested in GB (GMT <--> BST)

**Comment by chrisblakley** [comment]
Hey– this uses PHP's date functionality based on the timezone provided, so it should handle DST. That being said, I have not thoroughly tested various timezones myself– however improvements have been made over the years based on others' comments, so I think it is working much better for various timezones now. I can say that it has been working for Standard Time and DST for my timezone very well.

For reference, here is the bit of code that handles that:
https://github.com/chrisblakley/Daylight-Calendar-ICS/blob/master/daylight.php#L57-L71

I'm curious if you find that it handles it properly for you.

**Reply by Steve-Hight** [reply]
Hi Chris, thank you, I checked through the calendar, and yes the times move 1 hour forward and back on the correct days.
This is a great Calendar - many thanks
